ioutilx > copy - not working on Windows

```
Running tool: C:\Go\bin\go.exe test -coverprofile=C:\Users\Teamwork\AppData\Local\Temp\go-code-cover -timeout 30s github.com\teamwork\hub\api

# github.com/teamwork/hub/vendor/github.com/teamwork/utils/ioutilx
c:\golang\src\github.com\teamwork\hub\vendor\github.com\teamwork\utils\ioutilx\copy.go:223:32: undefined: syscall.Stat_t
Error: Tests failed.
```

Not sure how easy this fix is or if we should just give up on Windows 😄

--- a/ioutilx/copy.go
+++ b/ioutilx/copy.go
@@ -11,7 +11,6 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"syscall"
 	"time"
 
 	"github.com/pkg/errors"
@@ -220,13 +219,9 @@ func CopyMode(src, dst string, modes Modes) error {
 	}
 
 	if modes.Owner {
-		statT, ok := srcStat.Sys().(*syscall.Stat_t)
-		if !ok {
-			return errors.New("could not get file owner: type assertion to syscall.Stat_t failed")
-		}
-		err := os.Chown(dst, int(statT.Uid), int(statT.Gid))
+		err := setOwner(srcStat, dst)
 		if err != nil {
-			return errors.Wrap(err, "could not chown")
+			return err
 		}
 	}
 

--- a/ioutilx/copy_unix.go
+++ b/ioutilx/copy_unix.go
@@ -1,0 +1,23 @@
+// +build darwin dragonfly freebsd linux nacl netbsd openbsd solaris
+
+package ioutilx
+
+import (
+	"os"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+func setOwner(srcStat os.FileInfo, dst string) error {
+	statT, ok := srcStat.Sys().(*syscall.Stat_t)
+	if !ok {
+		return errors.New("could not get file owner: type assertion to syscall.Stat_t failed")
+	}
+	err := os.Chown(dst, int(statT.Uid), int(statT.Gid))
+	if err != nil {
+		return errors.Wrap(err, "could not chown")
+	}
+
+	return nil
+}

--- a/ioutilx/copy_windows.go
+++ b/ioutilx/copy_windows.go
@@ -1,0 +1,8 @@
+package ioutilx
+
+import "os"
+
+func setOwner(srcStat os.FileInfo, dst string) error {
+	// TODO: No-op
+	return nil
+}

